dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/CreateUserHelper.cs

```diff
@@ -87,6 +87,11 @@ public async Task<User> CreateUserWithTrnLookup(AuthenticationState authenticati
             TrnLookupStatus = authenticationState.TrnLookupStatus
         };
 
+        if (populatePreferredName)
```

Comment:
I do wonder if we need to restrict assigning preferred name to only the legacy TRN journey. Would be nice to not have to do this check

Reply:
this seems to imply it will be available in other journeys
https://trello.com/c/SFWbP5eY/1041-add-preferred-name-to-core-registration-journey